exo-go/features/run/app-dependency.feature

```diff
@@ -0,0 +1,23 @@
+Feature: running Exosphere applications with dependencies
```

Comment:
Yes, I am aware of that and don't hold it against you. We might have experienced some technical drift here. Since you reimplement the tests in Go, this is a good chance to also fine-tune the wording.